operator/controllers/execution/scans/job.go

```diff
@@ -29,7 +29,7 @@ func checkIfAllJobsCompleted(jobs *batch.JobList) jobCompletionType {
 	hasCompleted := true
 
 	for _, job := range jobs.Items {
-		if job.Status.Failed > 0 {
+		if job.Status.Failed > *job.Spec.BackoffLimit {
```

Comment:
What about input validation? What if one configurs "-42" in or "snafu" in the values.yaml?

Reply:
Then it would throw an exception because of a problem between keyboard and chair. 
But I can add a default value, if job.Spec.BackoffLimit is not a positive number. That enough?

Reply:
1. A default value makes sense, if the value is not configured or missed in values.yaml. Can this happen or is it this value somehow enforced?
2. Is the thrown exception to signal PEBCAK meaningful or is it a rather a FU-Exception which requires core-contributor skills to understand?

Reply:
Also backoffLimit is defined a follows:
```go
	// Specifies the number of retries before marking this job failed.
	// Defaults to 6
	// +optional
	BackoffLimit *int32 `json:"backoffLimit,omitempty" protobuf:"varint,7,opt,name=backoffLimit"`
```
"snafu" would not accepted:
```bash
$ helm install nmap ./
Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(ScanType.spec.jobTemplate.spec.backoffLimit): invalid type for io.securecodebox.execution.v1.ScanType.spec.jobTemplate.spec.backoffLimit: got "string", expected "integer"
```
-42 is accepted but does not affect the logic, it behaves as if it was 0

Reply:
Ok, both should be mentioned in the commit message bc it is implicit knowledge and future Ilyes will be thankful for this info ;-)